hibernate-java8/hibernate-java8.gradle

```diff
@@ -21,4 +21,10 @@ sourcesJar {
     // The OSGi JAR manifest support does not like a non-existent classes dir,
     // so make sure we dont use the OSGi one :)
     manifest = null
-}
\ No newline at end of file
+}
+
+javadocJar {
```

Comment:
I also realised that it might be better to _suppress_ the generation of JavaDoc for the deprecated module `hibernate-java8` (it's getting the JavaDoc for `hibernate-core`). I'll amend the PR to do this.